app/gui/src/controller.rs

```diff
@@ -9,6 +9,7 @@
 // === Export ===
 // ==============
 
+pub mod breadcrumbs;
```

Comment:
This should be placed inside `searcher`, as they are specifically searcher breadcrumbs (not graph breadcrumbs for instance).